### DISCUSSION TITLE
Post-processing Arguments (bitrate and framerate)

### DISCUSSION THREAD

**Discussion by Qwerty-Space** [streamlink/streamlink#6304] [Q&A]
### Description

Hello,

I noticed there's a "quality" option and a "transcode" option.  But I'm not sure they're what I'm looking for.
Once the stream is downloaded, I'd like to transcode with a reduced bitrate and framerate, programatically.

How can this be done?

Thank you.

**Comment by bastimeyer** [comment]
https://ffmpeg.org/

**Reply by Qwerty-Space** [reply]
> https://ffmpeg.org/

Does streamlink return the name of the downloaded file once it's finished?  I'll be calling it in a bash script.
